### PR TITLE
Add ESRGAN Upscaler (Closes #115)

### LINKED ISSUE #115: Add real ESRGAN upscaler to the upscaler executor
realESRGAN has an anime-finetuned upscaler while SWINIR was mainly trained on photographs and makes a mess of lineart. We should be allowed to select different upscalers when using the `upscale` endpoint.

### PR DESCRIPTION
**Pull request by AmericanPresidentJimmyCarter**
Built on top of the [clipseg PR](https://github.com/jina-ai/dalle-flow/pull/117), can be merged after that branch is merged or you can review together.

### CODE REVIEW

**Comment by AmericanPresidentJimmyCarter** [comment]
Notes:

- Adds RealESRGAN upscalers with various options to configure which to use
- Creates a user account in the clipseg/realesrgan to make passing a cache
  directory easier for the flow docker or native users
- Adds replica numbers as environmental options in flow_parser.py
- Corrects outdated version of jax that caused the docker to fail to load when
  attempting to use DALLE-MEGA

**Comment by AmericanPresidentJimmyCarter** [comment]
To use: 

```py
server_url = 'grpc://192.168.xxx.xxx:51005'
img = Image.open('man.jpg')
buffered = BytesIO()
img.save(buffered, format='PNG')
_d = Document(
    blob=buffered.getvalue(),
    mime_type='image/png',
).convert_blob_to_datauri()
da = DocumentArray([_d])

da[0].text = prompt

da = DocumentArray([da[0]]).post(f'{server_url}/realesrgan', parameters={
    'face_enhance': True,
    'model_name': 'RealESRGAN_x4plus',
})[0].matches

document_to_pil(da[0]).save('realesrgan_upscale.png')
```

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: executors/stable/config.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 jtype: StableDiffusionGenerator
 with:
-  batch_size: 1
+  batch_size: 4
```
[the remark this answers is not in the document]
Comment (AmericanPresidentJimmyCarter) [reply executors/stable/config.yml:3]: Sorry, I will revert. It was probably brought in by accident with a `git add .`.

File: executors/realesrgan/executor.py [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+class RealESRGANUpscaler(Executor):
+    # resrgan_models is organized as:
+    # {
+    #   model_name: {
+    #     'name': str,
+    #     'netscale': int, (scaling strength eg 4=4x)
+    #     'model': initialized RealESRGAN model,
+    #     'model_face_fix': initialized GFPGAN model, [optional, non-anime only]
+    #   }
+    # }
```
Comment (samsja) [inline executors/realesrgan/executor.py:42]: if not needed lets remove

**Review by samsja — COMMENTED** [review]
lgtm, few minor comments:

* could you run `black -s` formatter on your added code ?
* Lets add a tiny description of what the model is doing maybe in the docstring of the Executor. Have a link to the paper etcc

File: executors/realesrgan/executor.py (hunk 2 above)
Comment (AmericanPresidentJimmyCarter) [reply executors/realesrgan/executor.py:42]: It's a comment that explains how the `resrgan_models` dict entries are defined, so it's useful to anyone programming within the module.

**Comment by AmericanPresidentJimmyCarter** [comment]
@samsja Code has been added and a docstring added to the module. Should be ready to go.

**Review by samsja — APPROVED** [review]
lgtm

**Pull request merged**